apps/marginfi-v2-trading/src/components/common/Portfolio/PositionActionButtons.tsx

```diff
@@ -103,13 +107,15 @@ export const PositionActionButtons = ({
         throw new Error("Invalid client");
       }
 
```

Comment:
this function, should we also implement this in other places with prioFee? Currently set to 0 and added todo's

Reply:
where do you see it being set to 0? I can't find it in this file

Reply:
https://github.com/mrgnlabs/mrgn-ts/blob/977ab085401fc49855aba964933170cef83becf7/apps/marginfi-v2-trading/src/components/common/Pool/CreatePoolScript/components/CreatePoolLoading.tsx#L210 @k0beLeenders